CONTRIBUTING.rst

```diff
@@ -44,14 +44,23 @@ Implementation Report Process
 As of early 2020, the process to generate an EARL report for the official
 `JSON-LD Processor Conformance`_ page is:
 
+* Check out a local copy of the test suites::
+
+    > mkdir ../w3c
+    > cd ../w3c
+    > git clone https://github.com/w3c/json-ld-api.git
+    > git clone https://github.com/w3c/json-ld-framing.git
+    > cd ../pyld
+
+
 * Run the tests on the ``json-ld-api`` and ``json-ld-framing`` test repos to
   generate a ``.jsonld`` test report:
 
-  * ``python tests/runtests.py ../json-ld-api/tests/ ../json-ld-framing/tests/ -e pyld-earl.jsonld``
+  > ``python tests/runtests.py ..w3c/json-ld-api/tests/ ../w3c/json-ld-framing/tests/ -e pyld-earl.jsonld``
```

Comment:
As stated above, I'm not sure this w3c subdir idea is the best approach.  But if so, first path is missing a `/`.

Reply:
I'll leave it up to you how it should be resolved -- mostly it is just that I went directly to CONTRIBUTING and there wasn't anything that prevented me from interpreting it as asking for the tests to be run against the digitalbazaar tests instead of the W3C.